docs/extensibility/available-slot-fills.md

```diff
@@ -2,29 +2,37 @@
 
 This document presents the list of available Slots that you can use for adding your custom content (Fill):
 
-- [ExperimentalOrderMeta](#experimentalordermeta)
-- [ExperimentalOrderShippingPackages](#experimentalordershippingpackages)
-- [ExperimentalDiscountsMeta](#experimentaldiscountsmeta)
+-   [ExperimentalOrderMeta](#experimentalordermeta)
+-   [ExperimentalOrderShippingPackages](#experimentalordershippingpackages)
+-   [ExperimentalDiscountsMeta](#experimentaldiscountsmeta)
 
-***
+---
 
 If you want to add a new SlotFill component, check the [Checkout - Slot Fill document](../../packages/checkout/slot/README.md). To read more about Slot and Fill, check the [Slot and Fill document](./slot-fills.md).
 
 **Note About Naming:** Slots that are prefixed with `Experimental` are experimental and subject to change or remove. Once they graduate from the experimental stage, the naming would change and the `Experimental` prefix would be dropped.
 Check the [Feature Gating document](../blocks/feature-flags-and-experimental-interfaces.md) from more information.
 
-
 ## ExperimentalOrderMeta
+
 This Slot renders below the Checkout summary section and above the "Proceed to Checkout" button in the Cart.
 
-<img width="1135" alt="image" src="https://user-images.githubusercontent.com/6165348/118398683-a0202700-b651-11eb-8a4f-cd8b6ebff53f.png">
+Cart:
+
+<img width="865" alt="Screenshot 2022-02-17 at 16 42 33" src="https://user-images.githubusercontent.com/1628454/154517779-117bb4e4-568e-413c-904c-855fc3450dfa.png">
+
+Checkout:
+
+<img width="1233" alt="image" src="https://user-images.githubusercontent.com/6165348/118399133-90094700-b653-11eb-8ff0-c917947c199f.png">
```

Comment:
Is this the wrong image? I think you wanted to highlight the `OrderMeta` here.

Reply:
Yes, thank you for catching this. I've fixed it.